.github/action_scripts/check_clusters/metagraph.js

```diff
@@ -0,0 +1,55 @@
+const { clusterCheck } = require("./shared")
+
+const checkGlobalL0Node = async () => {
```

Comment:
This check repeats, can it be extracted and reused to not copy paste?

Reply:
Done